packages/dashboard/app/withdraw/utils/handlers.ts

```diff
@@ -0,0 +1,245 @@
+import { ethers } from 'ethers';
+import { PKPEthersWallet } from '@lit-protocol/pkp-ethers';
+import { TokenBalance, StatusType } from '../../../components/withdraw/types';
+import { sendEthTransaction, sendTokenTransaction, calculateEthGasCosts } from './transactionService';
+import { fetchERC20TokenBalances, fetchEthBalance } from './tokenUtils';
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+import { SessionSigs, IRelayPKP } from '@lit-protocol/types';
+import { SELECTED_LIT_NETWORK } from '@/components/consent/utils/lit';
+
+let isLitNodeClientInitialized = false;
+let litNodeClient: LitNodeClient;
+async function initLitNodeClient() {
+  if (!isLitNodeClientInitialized) {
+    litNodeClient = new LitNodeClient({
+      litNetwork: SELECTED_LIT_NETWORK
+    });
+    await litNodeClient.connect();
+    isLitNodeClientInitialized = true;
+  }
+  return litNodeClient;
+}
```

Comment:
Note: On the migration to vite let's move these things to providers, contexts and hooks. Check the [doc here](https://react.dev/learn/passing-data-deeply-with-context)